lib/Models/Cesium3DTilesCatalogItem.ts

```diff
@@ -60,332 +44,4 @@ export default class Cesium3DTilesCatalogItem
   get typeName() {
     return i18next.t("models.cesiumTerrain.name3D");
```

Comment:
I know this wasn't your change but perhaps fix while we're here :)

Reply:
Actually turns out thats the correct lang string, it's perhaps just a bit weirdly named

Reply:
Resolve later I think